Manito/Manito/Screens/Detail-Ing/DetailngCodebaseViewController.swift

```diff
@@ -294,12 +366,19 @@ final class DetailingCodebaseViewController: BaseViewController {
             $0.height.equalTo(80)
         }
         
-        view.addSubview(manitoOpenButton)
-        manitoOpenButton.snp.makeConstraints {
+        view.addSubview(manitoOpenButtonShadowView)
+        manitoOpenButtonShadowView.snp.makeConstraints {
             $0.bottom.equalTo(view.safeAreaLayoutGuide).inset(7)
             $0.centerX.equalToSuperview()
+            $0.width.equalTo(UIScreen.main.bounds.size.width - 40)
+            $0.height.equalTo(60.0)
         }
-
+        
+        manitoOpenButtonShadowView.addSubview(manitoOpenButton)
+        manitoOpenButton.snp.makeConstraints {
+            $0.top.leading.bottom.trailing.equalTo(manitoOpenButtonShadowView)
```

Comment:
p3;

```suggestion
            $0.edges.equalToSuperview()
```

`manitoOpenButton`이 manitoOpenButtonShadowView에 추가되었기 때문에 슈퍼뷰로 설정되었습니다 !
그리고 상하좌우는 `edges`로 깔끔하게 적을 수 있습니다 ^_______^

Reply:
> p3;
> 
> `manitoOpenButton`이 manitoOpenButtonShadowView에 추가되었기 때문에 슈퍼뷰로 설정되었습니다 ! 그리고 상하좌우는 `edges`로 깔끔하게 적을 수 있습니다 ^_______^

1afb7e6 오호~ 꿀팁 감사합니다~